packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -152,6 +175,38 @@ export class CfnInclude extends core.CfnElement {
     this.parameters[logicalId] = cfnParameter;
   }
 
+  private createOutput(logicalId: string): void {
+    const self = this;
+    const expression = new cfn_parse.CfnParser({
+      finder: {
+        findResource(lId: string): core.CfnResource | undefined {
+          if (!(lId in (self.template.Resources || {}))) {
```

Comment:
The body of `findResource()` here should be just `return self.resources[logicalId];`.